src/components/ModalVault.svelte

```diff
@@ -41,6 +41,17 @@
 
   $: selectedNote = resultNotes[selectedIndex]
   $: searchQuery = searchQuery ?? previousQuery
+  $: if (settings.openInNewPane) {
+    openInNewPaneKey = '↵'
```

Comment:
Since those variables are not declared before the reactive statement, they're implicitly global. See `ResultItemVault.svelte` how `title` is declared with a default value, then set from `settings.ignoreDiacritics`.

Note n this case you don't need a default value, since the `else` case is handled.

Reply:
Done